pkg/controller/wip/controller.go

```diff
@@ -342,7 +347,7 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 
 	servicePlan := findServicePlan(instance.Spec.PlanName, serviceClass.Plans)
 	if servicePlan == nil {
-		glog.Errorf("Instance %v/%v references a non-existent ServicePlan %v on ServiceClass %v", instance.Namespace, instance.Name, servicePlan.Name, serviceClass.Name)
+		glog.Errorf("Instance %v/%v references a non-existent ServicePlan %v on ServiceClass %v", instance.Namespace, instance.Name, instance.Spec.PlanName, serviceClass.Name)
```

Comment:
That's a fair point, but not related strictly to this PR.  One other thing that will happen, and has already happened in our codebase, is that everyone who adds a log statement that prints information about an object will do it differently.  When I was still working in the kubelet day to day, we had discussed having methods that gave you the printed representation for logs, and always using those so that the format was consistent everywhere and could be changed everywhere easily.

The only thing @vaikas-google is changing here is that there's a nil passed (the servicePlan, which couldn't be found) instead of meaningful information (the service plan named in the object's spec).  

I've created https://github.com/kubernetes-incubator/service-catalog/issues/451 for this